modules/installation-special-config-mirrored-disk.adoc

```diff
@@ -87,3 +87,70 @@ $ podman run --pull=always --rm --volume $HOME/clusterconfig:/pwd --workdir /pwd
 This command produces a combined Ignition config in `$HOME/clusterconfig/worker-raid.ign`.
 +
 . Continue with the remainder of the {product-title} deployment, using the combined Ignition config to provision nodes.
+
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume.
+
+. Create a machine config that configures a data volume by using software RAID.
+
+* To configure a data volume with RAID 1 on secondary disks, save the following machine config sample to a file in the `<installation_directory>/openshift` directory. For example, name the file `raid1-alt-storage.yaml`:
++
+.RAID 1 on secondary disks
+[source, yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: raid1-alt-storage
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      disks:
+      - device: /dev/sdc
+        partitions:
+        - label: data-1
+        wipeTable: true
+      - device: /dev/sdd
+        partitions:
+        - label: data-2
+        wipeTable: true
+      filesystems:
+      - device: /dev/md/md-var-lib-containers
+        format: xfs
+        path: /var/lib/containers
+        wipeFilesystem: true
+      raid:
+      - devices:
+        - /dev/disk/by-partlabel/data-1
+        - /dev/disk/by-partlabel/data-2
+        level: raid1
+        name: md-var-lib-containers
+    systemd:
+      units:
+      - contents: |-
+          [Unit]
+          Before=local-fs.target
+          Requires=systemd-fsck@dev-md-md\x2dvar\x2dlib\x2dcontainers.service
+          After=systemd-fsck@dev-md-md\x2dvar\x2dlib\x2dcontainers.service
```

Comment:
If the user changes the name of the RAID device, these need to be changed also, and they're kind of a pain with the escaping.  We can address this by avoiding implying a mountpoint path in the device name, so that users aren't tempted to change it.  Let's rename the RAID from `md-var-lib-containers` to `md-data` and update these two lines, the `What` line, and the filesystem `device` field.